smdebug/tensorflow/tensor_ref.py

```diff
@@ -4,6 +4,7 @@
 # Third Party
 import tensorflow.compat.v1 as tf
 from tensorflow.python.distribute import values
+from tensorflow.python.keras.mixed_precision.experimental import autocast_variable
```

Comment:
could this potentially fail in older TF versions?

Reply:
Support was added starting TF 2.1.
My latest commit addresses this.